content/administration/on_premise/source.rst

```diff
@@ -273,18 +273,23 @@ Dependencies
 
          .. tab:: Debian/Ubuntu
 
-            For Debian-based systems, the packages are listed in the `debian/control
-            <{GITHUB_PATH}/debian/control>`_ file of the Odoo sources.
```

Comment:
It could be nice to keep the href so one can review the list of packages from their browser.